### PR TITLE
BLOCKCHAIN

### PR DESCRIPTION
**Pull request by joaoachando**

### CODE REVIEW

**Review by sky-andremarques — APPROVED** [review]

**Review by sky-andremarques — CHANGES_REQUESTED** [review]
this is weird

**Review by miguelrnf — APPROVED** [review]

**Review by sky-andremarques — CHANGES_REQUESTED** [review]
why

**Comment by joaoachando** [comment]
> this is weird

why?

File: lib/blockchain/ContractLinking.dart [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+    List population = await _client
+        .call(contract: _contract, function: _tokens, params: []);
+    user = name[0];
+    tokens = population[0].toString();
+    print("$user , $tokens");
+    isLoading = false;
+    notifyListeners();
+  }
+
+  addData(String nameData, BigInt countData) async {
```
Comment (sky-andremarques) [inline lib/blockchain/ContractLinking.dart:90]: why

File: pubspec.lock [hunk 2]
```diff
@@ -1,13 +1,34 @@
...
+      url: "https://pub.dartlang.org"
+    source: hosted
+    version: "34.0.0"
+  analyzer:
+    dependency: transitive
+    description:
+      name: analyzer
+      url: "https://pub.dartlang.org"
+    source: hosted
+    version: "3.2.0"
```
Comment (sky-andremarques) [inline pubspec.lock:17]: what?

**Review by sky-andremarques — CHANGES_REQUESTED** [review]

File: pubspec.lock (hunk 2 above)
Comment (joaoachando) [reply pubspec.lock:17]: why

**Review by sky-andremarques — APPROVED** [review]

**Review by sky-miguelfranco — COMMENTED** [review]
??

**Pull request merged**